docs/Developers/Writing_Native_Bears.rst

```diff
@@ -20,6 +20,11 @@ directory of the coala-tutorial repository.
     If you want to wrap an already existing tool, please refer to
     :doc:`this tutorial instead<Writing_Linter_Bears>`.
 
+.. note::
+
+    Also refer `Python3 regex guide <https://docs.python.org/3/library/re.html>`_ for understanding regex.
+    For further information refer `this <http://www.diveintopython3.net/regular-expressions.html>`_.
```

Comment:
Imo this should be placed into writing-linter-bears under the regex-output-format section. This is kinda unrelated here.